index.js

```diff
@@ -141,44 +178,63 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+let ticketNumber = 0
+function takeATicketNumber(line){
+  ticketNumber++
+  line.push(ticketNumber)
+  return `Welcome. You are ticket number ${ticketNumber}`
+}
```

Comment:
awesome! Now you have  a variable defined in the global scope that you can alter with each invocation of this function.